src/lib/cozy-client/adapters/CozyStackAdapter.js

```diff
@@ -78,10 +91,14 @@ export default class CozyStackAdapter {
   }
 
   async fetchDocument(doctype, id) {
-    const doc = await cozy.client.data.find(doctype, id)
-    // we normalize again...
-    const normalized = { ...doc, id: doc._id, _type: doc._type }
-    return { data: [normalized] }
+    let doc
+    if (doctype === FILES_DOCTYPE) {
+      doc = normalizeDoc(await cozy.client.files.statById(id), doctype)
```

Comment:
Here you should pass `false` as a 2d arg to `statById` so that we're sure to hit the stack instead of the Pouch when an offline config for files exist (this is the stack adapter after all). See https://github.com/cozy/cozy-client-js/blob/master/src/files.js#L206